frontend/src/lib-customizations/espoo/citizen/assets/i18n/sv.tsx

```diff
@@ -271,7 +272,9 @@ const sv: Translations = {
       holidayFor: 'Semestertid för:',
       childOnHoliday: 'Barnet är på semester',
       addTimePeriod: 'Lägg till en period',
-      emptySelection: 'Ingen gratis ledighet'
+      emptySelection: 'Ingen gratis ledighet',
+      notEligible: (period: FiniteDateRange) =>
+        `Enkäten rör inte barnet, eftersom hen inte varit i småbarnspedagogiken oavbrutet under tiden ${period.start.format()} - ${period.end.format()}.`
```

Comment:
`period.formatCompact()` maybe?

Reply:
Didn't know of such a function! 😄 Changed to `period.format()` in this case.